packages/customer-effort-score/src/index.js

```diff
@@ -27,22 +33,53 @@ function CustomerEffortScore( {
 	label,
 } ) {
 	const [ score, setScore ] = useState( 0 );
-
-	if ( ! visible ) {
-		return null;
-	}
-
-	function close() {
-		setScore( 3 ); // TODO let this happen in the UI
-
-		toggleVisible();
+	const [ isOpen, setOpen ] = useState( visible );
+	const closeModal = () => {
+		setOpen( false );
+		toggleVisible( false );
+	};
+	const sendScore = () => {
+		closeModal();
 		trackCallback( score );
-	}
+	};
 
 	return (
-		<p className="customer-effort-score_modal">
-			{ label } <button onClick={ close }>Click me</button>
-		</p>
+		<>
+			{ isOpen && (
+				<Modal
+					className="woocommerce-customer-effort-score"
+					title="Please share your feedback"
```

Comment:
Text needs to be wrapped for translation. Use

```
import { __ } from '@wordpress/i18n';
```

Reply:
This is now fixed in 111472a652fc4693871aa76be21efe348b5a70fe.